extensions/amp-subscriptions/0.1/platform-store.js

```diff
@@ -468,18 +468,17 @@ export class PlatformStore {
    * @param {string} serviceId
    */
   reportPlatformFailure(serviceId) {
-    if (this.failedPlatforms_.indexOf(serviceId) == -1) {
+    if (serviceId === this.getLocalPlatform().getServiceId()
+      && this.fallbackEntitlement_) {
```

Comment:
What would happen if the `fallbackEntitlement` is successful, but other services have not finished yet? Will we execute the "local" renderer and send "local" pingback even though some other services might return with an actual positive response? Or does "renderer" and "pingback" wait for other services anyway?

Reply:
Well in theory its just filling local with a proxied Entitlement and doesn't change anything with `selectPlatform` flow. But will cross check and see if we have a test for that